C/common/Types.h

```diff
@@ -3,6 +3,11 @@
 
 typedef unsigned char uint8_t;
 typedef signed   char sint8_t;
+typedef unsigned short int  uint16;
+typedef signed short int    sint16;
```

Comment:
I will approve for this change for now but we need a discussion to decide what will we call these types <br>
1) Avoid linking with compiler libraries and call it `uint16_t` <br>
2)  Call all types `uint16` and `uint8`